src/lib/ui/thermostat/ClimatePresets.tsx

```diff
@@ -0,0 +1,196 @@
+  >(null)
+
+  const [inDeletionPresetKey, setInDeletionPresetKey] = useState<
+    Preset['climate_preset_key'] | null
+  >(null)
+  const deleteMutation = useDeleteThermostatClimatePreset()
+
+  if (
+    selectedClimatePreset != null ||
+    selectedClimatePreset === CreateNewPresetSymbol
+  ) {
+    return (
+      <ClimatePreset
+        onBack={() => {
+          setSelectedClimatePreset(null)
+        }}
+        device={device}
+        preset={
+          selectedClimatePreset === CreateNewPresetSymbol
+            ? undefined
+            : selectedClimatePreset
+        }
+      />
+    )
+  }
+
+  return (
+    <div className='seam-thermostat-climate-presets'>
+      <ContentHeader title='Climate Presets' onBack={onBack} />
+      <div className='seam-thermostat-climate-presets-body'>
+        <Button
+          onClick={() => {
+            setSelectedClimatePreset(CreateNewPresetSymbol)
+          }}
+          className='seam-climate-presets-add-button'
+        >
+          <AddIcon />
+          Create New
+        </Button>
+
+        <div className='seam-thermostat-climate-presets-cards'>
+          {device.properties.available_climate_presets.map((preset) => (
+            <PresetCard
+              onClickEdit={() => {
+                setSelectedClimatePreset(preset)
+              }}
+              onClickDelete={() => {
+                setInDeletionPresetKey(preset.climate_preset_key)
+                deleteMutation.mutate({
+                  climate_preset_key: preset.climate_preset_key,
+                  device_id: device.device_id,
+                })
+              }}
+              temperatureUnit={props.temperatureUnit}
+              preset={preset}
+              key={preset.climate_preset_key}
+              deletionLoading={
+                deleteMutation.isPending &&
+                inDeletionPresetKey === preset.climate_preset_key
+              }
+              disabled={
+                deleteMutation.isPending &&
+                inDeletionPresetKey !== preset.climate_preset_key
+              }
+            />
+          ))}
+        </div>
+      </div>
+    </div>
+  )
+}
+
+function PresetCard(
+  props: HTMLAttributes<HTMLDivElement> & {
+    preset: Preset
+    temperatureUnit: 'fahrenheit' | 'celsius'
+    onClickEdit: () => void
+    onClickDelete: () => void
+    deletionLoading?: boolean
+    disabled?: boolean
+  }
+): JSX.Element {
+  const {
+    preset,
+    temperatureUnit,
+    onClickEdit,
+    onClickDelete,
+    deletionLoading = false,
+    disabled = false,
+    ...attrs
+  } = props
+
+  const heatPoint =
+    temperatureUnit === 'fahrenheit'
+      ? preset.heating_set_point_fahrenheit
+      : (preset.heating_set_point_celsius ?? undefined)
+
+  const coolPoint =
+    temperatureUnit === 'fahrenheit'
+      ? preset.cooling_set_point_fahrenheit
+      : (preset.cooling_set_point_celsius ?? undefined)
+
+  const unitSymbol = temperatureUnit === 'fahrenheit' ? '˚F' : '˚C'
```

Comment:
Maybe this belongs in a more general `src/lib/seam/thermostats/units.ts` as `geTtemperatureUnitSymbol` along with the conversions function